app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -97,4 +87,8 @@ open class BpkSnapshotTest(private val tags: List<Any> = emptyList()) : Screensh
 
     return result
   }
+
+  private fun dpToPx(@Dimension dp: Int): Int {
```

Comment:
we have that helper function already in the backpack module, but it's internal. rather than exposing for the app declare it again here